src/main/java/com/coherent/training/selenium/kapitsa/web/pages/LoginPage.java

```diff
@@ -22,15 +21,16 @@
     private final By twitterIcon = By.xpath("//button[@aria-label='Twitter']");
 
     public LoginPage(WebDriver driver) {
-        this.driver = driver;
+        super(driver);
     }
 
     public MailBoxPage loginWithValidCredentials(String login, String pass){
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(20));
-        driver.findElement(loginField).sendKeys(login);
-        driver.findElement(loginButton).click();
-        driver.findElement(passField).sendKeys(pass);
-        driver.findElement(loginButton).click();
+
+        type(loginField, login);
+        clickOn(loginButton);
+        type(passField, pass);
+        clickOn(loginButton);
 
         return new MailBoxPage(driver);
```

Comment:
still - let's have this method as the composition of smaller methods like
this.enterCreds(login, pass);
this.clickOnLoginButton()
return new MailBoxPage(driver);